File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        maps={}
        for i,n in enumerate(nums):
            num=target-n
            if num in maps:
                return [maps[num],i]
            maps[n]=i
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& num, int target) {
        vector<int> result;
        for(int i=0;i<num.size();i++)
        {
            for(int j=i+1;j<num.size();j++)
            {
                if(num[i]+num[j]==target)
                {
                    result.push_back(i);
                    result.push_back(j);
                    break;
                }
            }
        }
        return result;
    }
};